extras/Dockerfile.qemu

```diff
@@ -0,0 +1,3 @@
+FROM fedora:31
+
+RUN dnf -y update nettle && dnf -y install iptables iproute dnsmasq qemu openssh-clients golang make && dnf clean all
```

Comment:
If you don't mind installing qemu from debian or ubuntu, I  can do that.

Reply:
In general I would expect that on the long run we would have different containers with different qemu versions, and a matrix of qemu tests. 

Just for your interest, I played with static qemu containers in the past (https://github.com/kubevirt/kubevirtci/pull/4/files#diff-26a5211abb5f6d55c0599070f481965c) and it would allow pretty small containers. However probably not an immediate topic.

Reply:
Agreed that in the long run we'll want to have a bigger matrix, I just thought we'd keep things simpler for now and base both on the same image. Actually, since the only real difference is whether or not QEMU is available in the container, we could simply replace the existing `Dockerfile` with this one and just run everything in the resulting `buildenv` container. How does that sound?

Reply:
sounds good. Will change it.

Reply:
Done.

Reply:
Could you build and push the tag and tell me the resulting image and tag?

Reply:
Done: `quay.io/virtblocks/buildenv:1.1.0`.